lib/App/Config/Chronicle.pm

```diff
@@ -4,18 +4,19 @@ package App::Config::Chronicle;
 use strict;
 use warnings;
 use Time::HiRes qw(time);
+use List::Util qw(any pairs pairmap);
 
 =head1 NAME
 
 App::Config::Chronicle - An OO configuration module which can be changed and stored into chronicle database.
 
 =head1 VERSION
 
-Version 0.05
+Version 0.07
```

Comment:
Actually, thanks for drawing my attention to this because I shouldn't be bumping the version number manually, that happens automatically during release.

I'll investigate how we can control the version that `dzil` sets.